uiowa/uiowa
# Adjust role names

Started looking into what it would take to accomplish https://github.com/uiowa/uiowa/issues/1905#issuecomment-1040446018 and ended up with this.

Resolves #5448 

# How to test

- [File review](https://github.com/uiowa/uiowa/pull/5447/files) to check that only the following changes are happening:
  - `administrator` becomes `developer`
  - `Administrator` becomes `Developer`
  - `webmaster` becomes `administrator`
  - `Webmaster` becomes `Administrator`
- `ddev blt ds`
- `ddev drush @default.local uli admin/people/permissions`
- Spot check permissions to see that match what is expected.
